### ISSUE TITLE
Question:  programing atmega4809 chips?

### ISSUE THREAD

**Issue by GlennWSo** [ElTangas/jtag2updi#71]
Hi,

In a project where im using atmega4809 in a custom pcb, i just broke my pickit 4 programmer.
I wonder if i can use the jtag2udpi in my Arduino Uno, as a replacement?

Question: can this firmware be used for programing atmega4809 chips?

**Comment by colinluthier** [comment]
Yes, it supports the 4808 and 4809. This video uses a nano but it shows the steps:
https://www.youtube.com/watch?v=yWb7qHI0YN4
